tools/react-app-template/src/md/full.ts

```diff
@@ -0,0 +1,5052 @@
+// (C) 2022 GoodData Corporation
+
+/* eslint-disable */
+/* THIS FILE WAS AUTO-GENERATED USING CATALOG EXPORTER; YOU SHOULD NOT EDIT THIS FILE; GENERATE TIME: 2022-11-29T15:39:32.596Z; */
+// @ts-ignore ignore unused imports here if they happen (e.g. when there is no measure in the workspace)
```

Comment:
Do we want this prepopulated? How will we keep it up to date with the backend state?

Reply:
I think we want, so you have everything ready to start and play.
Keep it up to date should be ok, if we will adopt the trial demo fixtures, they are not changed very often (and if we want some custom, these will be managed by ourselves, so we can put it for example to the release checklist)